### ISSUE TITLE
Unable to specify multiple columns to search

### ISSUE THREAD

**Issue by mattheyan** [DeuxHuitHuit/quicksearch#16]
Hi,

I'm using the following configuration, attempting to specify a sub-set of columns to search in.

```javascript
$("input[type=text]").quicksearch("table tbody tr:not(.header)", {
	selector: "td.searchable"
});
```

Since each row contains multiple searchable elements, the indices of the row cache and text cache do not match. This results in unexpected rows being returned.

For example:

| Name | Location |
| ------- | ---------- |
| Bob   | Brazil       |
| Joe   | Germany |
| Sara | England   |

If both columns match the "td.searchable" selector, and I search for "B", I would get back the first and second rows, because the first and second columns (both in row 1) match the filter.

I will submit a pull request if you are not opposed to supporting this scenario.

Thanks,
Bryan

**Comment by nitriques** [comment]
Yeah please send a PR !

Ideally, create a jsfiddle which illustrate the problem...

**Comment by mattheyan** [comment]
Here ya go
https://jsfiddle.net/xprjxv5j/

PR coming soon.

**Comment by nitriques** [comment]
Ooooh I see! Yes please send a PR !! Thanks!

**Comment by nitriques** [comment]
Fixed in #17 Thanks!
